Header header
#string identifying the corresponding element time 
string[] labels
#the execution time in seconds
float64[] times